packages/upload-client/src/sharding.js

```diff
@@ -22,21 +23,30 @@ export class ShardingStream extends TransformStream {
     let shard = []
     /** @type {import('@ipld/unixfs').Block[] | null} */
     let readyShard = null
-    let size = 0
+    let shardBlockLength = 0
```

Comment:
Nit: I was confused by the name while reading, I think `currentLength` or `byteLength` would be more intuitive.